cluster/gce/container-linux/configure-helper.sh

```diff
@@ -479,16 +482,16 @@ function start-kubelet {
     flags+=" --enable-debugging-handlers=false"
     flags+=" --hairpin-mode=none"
     if [[ "${REGISTER_MASTER_KUBELET:-false}" == "true" ]]; then
-      flags+=" --api-servers=https://${KUBELET_APISERVER}"
+      sed -i s/"example"/"${KUBELET_APISERVER}"/g /var/lib/kubelet/kubelet.kubeconfig
```

Comment:
mutating this file here seems really wrong, I'd prefer the file be generated with the right value.

if we absolutely can't generate the kubeconfig file with the correct hostname from the beginning, can we use kubectl to do this in a structured way:
```
kubectl config set-cluster local --server="https://${KUBELET_APISERVER}" --kubeconfig=/var/lib/kubelet/kubelet.kubeconfig
```